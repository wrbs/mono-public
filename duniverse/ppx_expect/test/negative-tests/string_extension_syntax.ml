let%expect_test "correction for expect node" =
  print_endline "Testing";
  print_endline "1";
  print_endline "2";
  print_endline "3";
  {%expect xxx|what|xxx}
;;

let%expect_test "correction for expect exact node" =
  print_endline "Testing";
  print_endline "1";
  print_endline "2";
  print_endline "3";
  {%expect_exact|what|}
;;
