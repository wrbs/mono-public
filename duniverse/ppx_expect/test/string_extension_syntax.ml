let%expect_test "correction for expect node" =
  print_endline "Testing";
  print_endline "1";
  print_endline "2";
  print_endline "3";
  {%expect|
    Testing
    1
    2
    3
    |}
;;

let%expect_test "correction for expect exact node" =
  print_endline "Testing";
  print_endline "1";
  print_endline "2";
  print_endline "3";
  {%expect_exact|Testing
1
2
3
|}
;;
