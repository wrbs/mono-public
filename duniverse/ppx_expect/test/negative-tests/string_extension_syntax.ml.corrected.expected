let%expect_test "correction for expect node" =
  print_endline "Testing";
  print_endline "1";
  print_endline "2";
  print_endline "3";
  {%expect xxx|
    Testing
    1
    2
    3
    |xxx}
;;

let%expect_test "correction for expect exact node" =
  print_endline "Testing";
  print_endline "1";
  print_endline "2";
  print_endline "3";
  {%expect_exact|Testing
1
2
3
|}
;;
